
<!DOCTYPE html>


<html lang="en" data-content_root="../../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Question 2: “What could go wrong?” &#8212; NCCoE Genomic Data Threat Modeling Project</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-design.min.css?v=95c83b7e" />
    <link rel="stylesheet" type="text/css" href="../../_static/content.css?v=59cd0ab0" />
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/2.1.8/css/dataTables.dataTables.min.css" />
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/2.1.8/css/dataTables.bootstrap5.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/custom.css?v=60ee0719" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../../_static/jquery.js?v=5d32c60e"></script>
    <script src="../../_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
    <script src="../../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../../_static/doctools.js?v=9a2dae69"></script>
    <script src="../../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../../_static/copybutton.js?v=f281be69"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../../_static/togglebutton.js?v=4a39c7ea"></script>
    <script src="../../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script src="../../_static/design-tabs.js?v=f930bc37"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'Vol_C/PTM/Question2';</script>
    <script src="../../_static/jquery.visible.js?v=5b129ffc"></script>
    <script src="../../_static/jquery.leaveNotice-nist.js?v=7cf1efa2"></script>
    <script src="../../_static/applyLeaveNotice.js?v=ca661b5e"></script>
    <script src="../../_static/smoothNavScroll.js?v=2ce98a12"></script>
    <script src="https://cdn.datatables.net/2.1.8/js/dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/2.1.8/js/dataTables.bootstrap5.min.js"></script>
    <script src="../../_static/main.js?v=74083827"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Question 3: “What are we going to do about it?”" href="Question3.html" />
    <link rel="prev" title="Question 1: “What are we working on?”" href="Question1.html" />
    
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>

    <!-- Google tag (gtag.js) -->
    <script async
        src="https://www.googletagmanager.com/gtag/js?id=G-RJSMY46M5C"></script>
    <script>
        window.dataLayer = window.dataLayer || [];
        function gtag() { dataLayer.push(arguments); }
        gtag('js', new Date());
        gtag('config', 'G-RJSMY46M5C');

    </script>
    <!-- Old DAP UA tag -->
    <script async type="text/javascript" id="_fed_an_ua_tag"
        src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=NIST&subagency=github&pua=UA-66610693-1&yt=true&exts=ppsx,pps,f90,sch,rtf,wrl,txz,m1v,xlsm,msi,xsd,f,tif,eps,mpg,xml,pl,xlt,c">
    </script>
    <link rel="stylesheet" href="https://pages.nist.gov/nist-header-footer/css/nist-combined.css">
    <link rel="stylesheet" type="text/css" href="https://pages.nist.gov/leaveNotice/css/jquery.leaveNotice.css" />
    <script src="https://pages.nist.gov/nist-header-footer/js/nist-header-footer-v-2.0.js" type="text/javascript" defer="defer"></script>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search..."
         aria-label="Search..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../../index.html">
  
  
  
  
  
  
    <p class="title logo__title">NCCoE Genomic Data Threat Modeling Project</p>
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../../index.html">
                    Genomic Data Threat Modeling
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../../Vol_A/ExecutiveSummary.html">Executive Summary</a></li>
<li class="toctree-l1 current active has-children"><a class="reference internal" href="../index.html">Genomic Data Threat Modeling: Privacy</a><details open="open"><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="../Summary.html">Summary</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Introduction.html">Introduction</a></li>
<li class="toctree-l2 current active has-children"><a class="reference internal" href="index.html">Genomic Data Threat Modeling Example</a><details open="open"><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="Question1.html">Question 1: "What are we working on?"</a></li>
<li class="toctree-l3 current active"><a class="current reference internal" href="#">Question 2: "What could go wrong?"</a></li>
<li class="toctree-l3"><a class="reference internal" href="Question3.html">Question 3: "What are we going to do about it?"</a></li>
<li class="toctree-l3"><a class="reference internal" href="Question4.html">Question 4: "How did we do?"</a></li>
</ul>
</details></li>
<li class="toctree-l2"><a class="reference internal" href="../Conclusion.html">Conclusion</a></li>
<li class="toctree-l2 has-children"><a class="reference internal" href="../Appendix/index.html">Appendices</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixA.html">Appendix A</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixB.html">Appendix B</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixC.html">Appendix C</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixD.html">Appendix D</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixE.html">Appendix E</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixF.html">Appendix F</a></li>
<li class="toctree-l3"><a class="reference internal" href="../Appendix/appendixG.html">Appendix G</a></li>
</ul>
</details></li>
</ul>
</details></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../../_sources/Vol_C/PTM/Question2.rst" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.rst</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Question 2: “What could go wrong?”</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#linddun-analysis">LINDDUN Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#panoptic-analysis">PANOPTIC Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#threat-validation">Threat Validation</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section id="question-2-what-could-go-wrong">
<h1>Question 2: “What could go wrong?”<a class="headerlink" href="#question-2-what-could-go-wrong" title="Link to this heading">#</a></h1>
<p>At this point environmental and system context have been captured and the operational dataflows and actions have been documented. The analytical processes of genomic data threat modeling for privacy must now be applied to these descriptions. Those processes consist of two principal activities: (1) dataflow analysis to identify threats and (2) threat alignment and validation.</p>
<p>To address “what could go wrong,” the dataflow analysis (1) employed LINDDUN and its catalog of threat trees to associate potential genomic data threats regarding privacy with specific dataflows and actions, then (2) created PANOPTIC attack mappings for the genomic sequencing workflow. Both models are needed because the LINDDUN analysis identifies abstract threats that are theoretically possible while PANOPTIC identifies steps that could form a practical attack. Where practical attack and theoretical threat align, the combination is validated against the NIST PEOs. This exercise ensures that potential threats are both conceivable and executable, and that these would impact at least one of the NIST PEOs.</p>
<section id="linddun-analysis">
<h2>LINDDUN Analysis<a class="headerlink" href="#linddun-analysis" title="Link to this heading">#</a></h2>
<p>The LINDDUN <a class="reference internal" href="../Appendix/appendixB.html#ref2" id="id1"><span>[Ref2]</span></a> methodology involves assessing each distinct dataflow for potential threats. A dataflow consists of a source, the flow itself, and a destination. To avoid confusion, we refer to this triad as a dataflow segment. Using the modified Assess System Design table in PRAM Worksheet 2, each dataflow segment in the core example DFD (Figure 4) is documented. In addition to the source, flow, and destination, the applicable data actions[12]_ are also noted. Each dataflow segment is also assigned a purpose (based on the Data Action Key) using the Context column.</p>
<p>For each documented dataflow segment, relevant LINDDUN threats are then identified, using as a starting point the mapping of segment-based high-level threat types, shown in Table 11. This mapping is a heuristic for determining potential LINDDUN threats and involved components. The LINDDUN threat trees[Ref2]_ that detail those threat types can then be used to determine whether and which more granular threats potentially apply to that segment based on its constituent elements and context. Those threats judged potentially applicable are captured in the LINDDUN Analysis column in Table 12, including the scenario. Note that multiple threats may apply to a single dataflow segment.</p>
<div class="pst-scrollable-table-container"><table class="table" id="id8">
<caption><span class="caption-text">Table 11. LINDDUN Per Element Threat Mapping Heuristic</span><a class="headerlink" href="#id8" title="Link to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="head"><p><strong>Source (Src)</strong></p></th>
<th class="head"><p><strong>Destination (Dst)</strong></p></th>
<th class="head"><p><strong>L</strong></p></th>
<th class="head"><p><strong>I</strong></p></th>
<th class="head"><p><strong>NR</strong></p></th>
<th class="head"><p><strong>D</strong></p></th>
<th class="head"><p><strong>DD</strong></p></th>
<th class="head"><p><strong>U</strong></p></th>
<th class="head"><p><strong>NC</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Process</p></td>
<td><p>Process</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-Dst</p></td>
<td><p>Src-Dst</p></td>
</tr>
<tr class="row-odd"><td><p>Process</p></td>
<td><p>Store</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-Dst</p></td>
<td><p>Src-Dst</p></td>
</tr>
<tr class="row-even"><td><p>Process</p></td>
<td><p>External</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-Dst</p></td>
<td><p>Src-Dst</p></td>
</tr>
<tr class="row-odd"><td><p>Store</p></td>
<td><p>Process</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-Dst</p></td>
<td><p>Src-Dst</p></td>
</tr>
<tr class="row-even"><td><p>External</p></td>
<td><p>Process</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-flow</p></td>
<td><p>Src-flow-Dst</p></td>
<td><p>Src-Dst</p></td>
<td><p>Dst</p></td>
</tr>
</tbody>
</table>
</div>
<p>To illustrate, consider dataflow segment Number 1 in Table 12. It consists of a receiving clerk delivering a physical biological sample to a lab technician for genomic sequencing. Leveraging the data action column helps us infer that this is a process-to-process segment. Consulting Table 11 and the threat definitions, as well as the context of the segment, we conclude that linking is the only relevant threat. Sending samples to a technician known to be associated with work on a particular disease could link the samples to that disease, an instance of L.2.2.1, profiling an individual. The other possibilities can be dismissed because at this stage:</p>
<ul class="simple">
<li><p>The sample must still be associated with the direct data subject as part of the workflow</p></li>
<li><p>There is nothing for the direct data subject to repudiate, aside from providing the sample to those who must necessarily be aware that the sample has been provided</p></li>
<li><p>Because the sample must be identifiable, detection is unavoidable</p></li>
<li><p>The only data disclosure is inherent in the workflow and therefore unproblematic</p></li>
<li><p>The direct data subject has provided informed consent and is aware of their options</p></li>
<li><p>Standard practices are being employed in the workflow</p></li>
</ul>
<p>The process proceeds similarly for the remaining eight dataflow segments, resulting in Table 12. Note that a segment can be subject to more than one threat, as is the case for segment 8.</p>
<div class="pst-scrollable-table-container"><table class="table" id="id9">
<caption><span class="caption-text">Table 12. LINDDUN Dataflow Analysis for the Core Example</span><a class="headerlink" href="#id9" title="Link to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="head"><p><strong>No.</strong></p></th>
<th class="head"><p><strong>Source</strong></p></th>
<th class="head"><p><strong>Dataflow Type</strong></p></th>
<th class="head"><p><strong>Data Action 1</strong></p></th>
<th class="head"><p><strong>Data Action 2</strong></p></th>
<th class="head"><p><strong>Destination</strong></p></th>
<th class="head"><p><strong>Context (purpose)</strong></p></th>
<th class="head" colspan="2"><p><strong>LINDDUN</strong> <strong>Analysis</strong> <strong>(applicable</strong> <strong>threats)</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>1</p></td>
<td><p>Receiving Clerk (S1-PH)</p></td>
<td><p>Physical Sample</p></td>
<td><p>Transfer</p></td>
<td></td>
<td><p>Lab Tech (S2-A)</p></td>
<td><p>Send physical sample to lab tech for research project</p></td>
<td><p>L.2.2.1</p></td>
<td><p>Sending samples to wet lab known to be researching a specific disease at that time could link samples to that disease</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>Lab Tech (S2-A)</p></td>
<td><p>Physical Sample</p></td>
<td><p>Transfer</p></td>
<td></td>
<td><p>Wet Lab (S3-PH)</p></td>
<td><p>Send physical sample to wet lab for sequencing</p></td>
<td><p>L.2.2.1</p></td>
<td><p>Sending samples to wet lab known to be researching a specific disease at that time could link samples to that disease</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>Wet Lab (S3-PH)</p></td>
<td><p>Physical Sample</p></td>
<td><p>Transfer</p></td>
<td><p>Retention</p></td>
<td><p>Physical Sample Storage (S11-PH)</p></td>
<td><p>Send physical sample for storage in appropriate freezers</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Sending group of X samples together to freezers around the same time as a project known to be doing Y disease research could link the samples to Y disease</p></td>
</tr>
<tr class="row-odd"><td><p>4</p></td>
<td><p>Wet Lab (S3-PH)</p></td>
<td><p>Sample Metadata</p></td>
<td><p>Generation</p></td>
<td><p>Retention</p></td>
<td><p>LIMS (S4-PH)</p></td>
<td><p>Generate pseudonymized ID to be used for sample</p></td>
<td><p>I.2.1.1</p></td>
<td><p>Nature of genomic data makes complete disassociability impossible to guarantee</p></td>
</tr>
<tr class="row-even"><td><p>5</p></td>
<td><p>LIMS (S4-PH)</p></td>
<td><p>Sample Metadata</p></td>
<td><p>Transfer</p></td>
<td></td>
<td><p>Wet Lab (S3-PH)</p></td>
<td><p>Send back to wet lab the pseudonymized ID to be used for sample</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples put into LIMS around same time could receive IDs with linkable characteristics, which then allows linkage of sample group to a study around same time, unless LIMS is cautious of this</p></td>
</tr>
<tr class="row-odd"><td><p>6</p></td>
<td><p>Wet Lab (S3-PH)</p></td>
<td><p>Sequence Data</p></td>
<td><p>Transfer</p></td>
<td><p>Retention</p></td>
<td><p>Cluster Filesystem (S6-A)</p></td>
<td><p>Send digital sequence data to be stored</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples that are put into the cluster filesystem around the same time could be interpreted as being linked to a study about Y disease around the same time</p></td>
</tr>
<tr class="row-even"><td><p>7</p></td>
<td><p>Cluster Filesystem (S6-A)</p></td>
<td><p>Sequence Data</p></td>
<td><p>Transfer</p></td>
<td></td>
<td><p>Compute Nodes (S5-A)</p></td>
<td><p>Send digital sequence data to Compute Nodes to operate on digital sequence data to transform it into objective-specific data</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples sent to compute nodes around same time could be interpreted as being linked to a study about Y disease around same time</p></td>
</tr>
<tr class="row-odd"><td rowspan="2"><p>8</p></td>
<td rowspan="2"><p>Compute Nodes (S5-A)</p></td>
<td rowspan="2"><p>Sequence Data, Context-relevant Research Data</p></td>
<td rowspan="2"><p>Transformation</p></td>
<td rowspan="2"></td>
<td rowspan="2"><p>Cluster Filesystem (S6-A)</p></td>
<td rowspan="2"><p>Operate on sequence data to create context-relevant research data</p></td>
<td><p>DD.4.1.2</p></td>
<td><p>Bioinformatics tools come from a variety of developers that can change over time; corruption within this supply chain, especially if left unmonitored, could result in research subject data being disclosed</p></td>
</tr>
<tr class="row-even"><td><p>U.1.1</p></td>
<td><p>Data subject does not clearly understand what data actions that analysis tools along the pipeline will perform on their data</p></td>
</tr>
<tr class="row-odd"><td><p>9</p></td>
<td><p>Cluster Filesystem (S6-A)</p></td>
<td><p>Context-relevant Research Data</p></td>
<td><p>Transfer</p></td>
<td></td>
<td><p>Data Delivery DMZ (S13-A)</p></td>
<td><p>Send generated context-relevant research data to data delivery DMZ for to make it available for delivery</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples that are put into the data delivery DMZ around the same time could be interpreted as being linked to a study about Y disease around the same time</p></td>
</tr>
</tbody>
</table>
</div>
<p>The complete LINDDUN analysis can be found in <a class="reference external" href="Appendix/appendixE.html">Appendix E</a>. Note that for manageability the analysis was initially divided into clinical, research, and shared use cases, the last based on the common portion of the two use cases. The results were then combined into a single system design table. This table was then sorted on the specific LINDDUN threats.</p>
</section>
<section id="panoptic-analysis">
<h2>PANOPTIC Analysis<a class="headerlink" href="#panoptic-analysis" title="Link to this heading">#</a></h2>
<p>The LINDDUN analysis identifies potential threats at the level of dataflows. However, real-world privacy attacks are not typically launched at that level, nor do they consist of a single self-contained element. They are less abstract and operate at the system level. The PANOPTIC analysis is a necessary complement to the LINDDUN analysis as it will describe potential threats from a system perspective. The LINDDUN analysis is then used to determine whether the threats identified at the dataflow level support the projected attacks as described by PANOPTIC. If not, the PANOPTIC attacks are considered non-actionable.</p>
<p>While the LINDDUN analysis is grounded in system specifics as captured by DFDs, the PANOPTIC analysis involves actively imagining in practical terms what might take place. Utilizing the PANOPTIC Privacy Activities mapping template, a privacy attack mapping for the core example was generated. Table 13 lists the threat actions identified for the core example based on high-level knowledge of the system and its context. The complete PANOPTIC mappings for the clinical and research use cases are provided in <a class="reference external" href="Appendix/appendixE.html">Appendix E</a>.</p>
<div class="pst-scrollable-table-container"><table class="table" id="id10">
<caption><span class="caption-text">Table 13. Threat Actions Identified by the PANOPTIC Privacy Activity Mapping for the Core Example</span><a class="headerlink" href="#id10" title="Link to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="head"><p><strong>PANOPTIC Threat Action</strong></p></th>
<th class="head"><p><strong>Definition</strong></p></th>
<th class="head"><p><strong>Elaboration</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>PA02.02 Consent: Imprecise</p></td>
<td><p>Key data actions are not presented clearly enough to constitute informed consent</p></td>
<td><p>May not provide details on how research is conducted, and which parts of the pipeline are privacy-relevant</p></td>
</tr>
<tr class="row-odd"><td><p>PA03.09 Collection: Recording</p></td>
<td><p>Capturing a physical or digital artifact representing an aspect or likeness of the data subject</p></td>
<td></td>
</tr>
<tr class="row-even"><td><p>PA03.11 Collection: Biological sample</p></td>
<td><p>Collecting biological materials or specimens (e.g., blood, urine, tissue cells, or saliva) from the data subject</p></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>PA05.01.01 Identification:
Re-identification</p></td>
<td><p>Re-associating data with the data subject that had been treated to remove those associations</p></td>
<td></td>
</tr>
<tr class="row-even"><td><p>PA05.02.02 Identification: Pseudo-identifier</p></td>
<td><p>Assigning a pseudo-identifier (e.g., randomly generated ID)</p></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>PA07.01 Manageability: No individual access to information</p></td>
<td><p>The data subject or their proxy cannot obtain or view their collected personal data</p></td>
<td></td>
</tr>
<tr class="row-even"><td><p>PA07.02 Manageability: No individual management of information content</p></td>
<td><p>The data subject or their proxy cannot transform (e.g., move, copy, edit) their collected personal data</p></td>
<td><p>Direct data subject cannot change their data that is used for research</p></td>
</tr>
<tr class="row-odd"><td><p>PA07.03 Manageability: No individual deletion of information</p></td>
<td><p>The data subject or their proxy cannot delete their collected personal data</p></td>
<td><p>Once the research data is published, the direct data subject cannot remove theirs from the body of research</p></td>
</tr>
<tr class="row-even"><td><p>PA07.05 No individual control of information use</p></td>
<td><p>The data subject or their proxy cannot control how their information is used</p></td>
<td><p>Direct data subject cannot manage what types of research studies use their data</p></td>
</tr>
<tr class="row-odd"><td><p>PA08.01.01 Aggregation: Single source profiling</p></td>
<td><p>Assembling and organizing data points about specific data subjects from a single source</p></td>
<td><p>The research project must determine whether or not a given direct data subject exhibits the trait being studied, implying profiling with the single source being their provided sample</p></td>
</tr>
<tr class="row-even"><td><p>PA08.02.01 Aggregation: Single source clustering</p></td>
<td><p>Assembling and organizing data points regarding groups of people from a single source</p></td>
<td><p>Research studies may look for commonalities across genomic samples</p></td>
</tr>
<tr class="row-odd"><td><p>PA08.02.02 Aggregation:
Multi-source clustering</p></td>
<td><p>Assembling and organizing data points regarding groups of people from multiple sources</p></td>
<td><p>Research studies may seek insights on a specific population potentially characterized along multiple dimensions, implying clustering</p></td>
</tr>
<tr class="row-even"><td><p>PA09.01.01 Processing: Deriving information about individuals</p></td>
<td><p>Determining or extracting novel information about the data subject by analyzing information</p></td>
<td><p>Research project must determine if the trait being studied is exhibited by the data subject</p></td>
</tr>
<tr class="row-odd"><td><p>PA09.01.02 Processing: Deriving aggregate information</p></td>
<td><p>Determining or extracting novel aggregate information by analyzing information</p></td>
<td><p>Research project may seek insights about a given population regarding a genetic trait</p></td>
</tr>
<tr class="row-even"><td><p>PA09.01.03 Processing: Deriving sensitive information</p></td>
<td><p>Determining or extracting novel sensitive information by analyzing information</p></td>
<td><p>Genetic information and insights gained can be sensitive information</p></td>
</tr>
<tr class="row-odd"><td><p>PA09.01.04 Processing: Deriving derogatory information</p></td>
<td><p>Determining or extracting novel derogatory information by analyzing information</p></td>
<td><p>Genetic diseases or susceptibility to them can be considered derogatory information</p></td>
</tr>
<tr class="row-even"><td><p>PA09.03 Processing: Introducing bias</p></td>
<td><p>Data action is adversely influenced by bias</p></td>
<td><p>Bias could be introduced into research projects if the demographic spread of the data pool is not balanced. (This may not be possible for some studies, such as one targeting a trait only present in a specific population.)</p></td>
</tr>
<tr class="row-odd"><td><p>PA10.01 Sharing: Affording revelations</p></td>
<td><p>Making available information that enables the discovery of further information</p></td>
<td><p>A research project that a direct data subject joins may yield results now or in the future, including the relevance of the research topic for the data subject</p></td>
</tr>
<tr class="row-even"><td><p>PA11.01 Use: Implication</p></td>
<td><p>Establishing a particularized derogatory suspicion or accusation regarding the data subject</p></td>
<td></td>
</tr>
<tr class="row-odd"><td><p>PA12.01 Retention &amp; destruction: Data not destroyed after use</p></td>
<td><p>Information has not been disposed at the conclusion of its life cycle</p></td>
<td><p>May be indeterminate for research data</p></td>
</tr>
<tr class="row-even"><td><p>PA12.02 Retention &amp; destruction: Data improperly destroyed</p></td>
<td><p>Information remains at least partially recoverable despite attempts to destroy it</p></td>
<td><p>Flow cell insufficiently cleaned and sequencer supply chain not cleaning hard drives</p></td>
</tr>
</tbody>
</table>
</div>
<p>Table 14 describes five attack scenarios that are specific to the core example. Each scenario was determined by considering how specific threat actions could be used by an actor as part of an attack involving a distinct DFD segment. Since attacks could apply to different DFD segments, the table in some cases associates multiple identical attacks with the same scenario. <a class="reference external" href="Appendix/appendixF.html">Appendix F</a> provides the comprehensive analysis that was performed on the complete example, which includes all the Attack Numbers and Scenario IDs. Table 14 extracts only the attack scenarios relevant to the core example, aligning with the Attack Numbers, Scenario IDs, and Privacy Threat Actions from the comprehensive analysis found in <a class="reference external" href="Appendix/appendixF.html">Appendix F</a>.</p>
<div class="pst-scrollable-table-container"><table class="table" id="id11">
<caption><span class="caption-text">Table 14. Attack Scenarios Relevant to the Core Example</span><a class="headerlink" href="#id11" title="Link to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="head"><p><strong>Attack Numbers from Complete Example</strong></p></th>
<th class="head"><p><strong>Scenario ID</strong></p></th>
<th class="head"><p><strong>PANOPTIC Threat Actions Describing the Attack</strong></p></th>
<th class="head"><p><strong>Scenario Description</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>1, 14, 15</p></td>
<td><p>S1.1</p></td>
<td><p>PA03.09, PA03.11, PA08.01.01, PA10.01, PA11.01</p></td>
<td><p>Pipeline actor uses physical access to correlate study details with physical samples and associated metadata.</p></td>
</tr>
<tr class="row-odd"><td><p>2-5</p></td>
<td><p>S1.2</p></td>
<td><p>PA03.09, PA05.02.02, PA08.02.02, PA10.01, PA11.01</p></td>
<td><p>Pipeline actor uses physical access to correlate study details with digital data.</p></td>
</tr>
<tr class="row-even"><td><p>26</p></td>
<td><p>S6</p></td>
<td><p>PA05.01.01</p></td>
<td><p>Pipeline actor uses digital access to correlate study details with digital data.</p></td>
</tr>
<tr class="row-odd"><td><p>55</p></td>
<td><p>S6</p></td>
<td><p>PA03.09, PA09.01.01, PA09.01.03, PA09.01.04, PA11.01</p></td>
<td><p>Pipeline actor uses digital access to correlate study details with digital data.</p></td>
</tr>
<tr class="row-even"><td><p>65</p></td>
<td><p>S17</p></td>
<td><p>PA02.02, PA07.05</p></td>
<td><p>Sequencing service staff utilizes third party tools and software that may perform additional data actions unbeknownst to a direct data subject. <a class="footnote-reference brackets" href="#id7" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>13<span class="fn-bracket">]</span></a></p></td>
</tr>
</tbody>
</table>
</div>
<p>In the first scenario described in Table 14, attack numbers 1, 14, and 15, which constitute health status inference attacks, can be broken down as follows: The attack involves an actor with a role in the sequencing pipeline physically accessing artifacts relating to direct data subjects (PA03.09, Collection: Recording) in the form of biological samples (PA03.11) and their associated metadata (as per PC05). The actor can correlate the research studies that will use these samples with the samples and their metadata (PA08.01.01, Aggregation: Profiling: Single source profiling), which may reveal other information, such as potential susceptibility to a particular disease (PA10.01, Sharing: Affording revelations). This would enable the attacker to discern something negative about the individual’s health status (PA11.01, Use: Implication).</p>
</section>
<section id="threat-validation">
<h2>Threat Validation<a class="headerlink" href="#threat-validation" title="Link to this heading">#</a></h2>
<p>As previously indicated, threat validation consists of two steps: mapping PANOPTIC attacks to relevant LINDDUN threats and mapping LINDDUN-validated attacks against the NIST PEOs of predictability, manageability, and disassociability. If a PANOPTIC attack does not align with one or more LINDDUN threats or if an aligned attack does not appear to undermine at least one of the PEOs, then the threat is invalid and removed from further consideration during this modeling process iteration.</p>
<p>Validation of PANOPTIC attacks against LINDDUN threats amounts to assessing the relationship between the threat actions that constitute the attack and the relevant LINDDUN threats. In most cases, that relationship is many-to-many. Therefore, carrying out this assessment involves judgement informed by the surrounding context. To facilitate this determination, <a class="reference external" href="Appendix/appendixG.html">Appendix G</a> includes a mapping between PANOPTIC threat actions and LINDDUN threats in both directions. Because such mappings exist in all cases, the mere existence of a potentially relevant LINDDUN threat is insufficient validation.</p>
<p>For attacks aligned with LINDDUN threats, validation against the PEOs serves to confirm that the attacks actually met the definition of a threat put forward in Section 1.4 by potentially undermining system predictability, manageability, and/or disassociability. Some attacks may impact more than one PEO, but a validated attack must impact at least one.</p>
<p>Table 15 lists the validation results for the five attack scenarios relevant to the <em>core</em> example from Table 14. These were extracted from the complete combined validation table found in <a class="reference external" href="Appendix/appendixG.html">Appendix G</a>. This table documents the LINDDUN Analysis and PEOs impacted by the threat, aligned to the Attack Number, the Scenario ID, PANOPTIC Threat Action, and LINDDUN Threat.</p>
<div class="pst-scrollable-table-container"><table class="table" id="id12">
<caption><span class="caption-text">Table 15. Core Example Attack Validations</span><a class="headerlink" href="#id12" title="Link to this table">#</a></caption>
<thead>
<tr class="row-odd"><th class="head"><p><strong>Attack Number</strong></p></th>
<th class="head"><p><strong>Scenario ID</strong></p></th>
<th class="head"><p><strong>PANOPTIC Threat Action</strong></p></th>
<th class="head"><p><strong>LINDDUN Threat</strong></p></th>
<th class="head"><p><strong>LINDDUN Analysis</strong></p></th>
<th class="head"><p><strong>Impacted PEOs</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>1</p></td>
<td><p>S1.1</p></td>
<td><p>PA03.09, PA03.11, PA08.02.01, PA10.01, PA11.01</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Sending the group of X samples together to the freezers around the same time as a project known to be doing Y disease research could link the samples to Y disease</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>S1.2</p></td>
<td><p>PA03.09, PA05.02.02, PA08.02.02, PA10.01, PA11.01</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples that are put into the LIMS around the same time could receive IDs with linkable characteristics, which then allows linkage of the sample group to a study around the same time, unless the LIMS implements measures to prevent this</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>S1.2</p></td>
<td><p>PA03.09, PA05.02.02, PA08.02.02, PA10.01, PA11.01</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples that are put into the cluster filesystem around the same time could be interpreted as being linked to a study about Y disease around the same time</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-odd"><td><p>4</p></td>
<td><p>S1.2</p></td>
<td><p>PA03.09, PA05.02.02, PA08.02.02, PA10.01, PA11.01</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples sent to the compute nodes around the same time could be interpreted as being linked to a study about Y disease around the same time</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-even"><td><p>5</p></td>
<td><p>S1.2</p></td>
<td><p>PA03.09, PA05.02.02, PA08.02.02, PA10.01, PA11.01</p></td>
<td><p>L.2.1.2</p></td>
<td><p>Samples that are put into the data delivery DMZ around the same time could be interpreted as being linked to a study about Y disease around the same time</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-odd"><td><p>14</p></td>
<td><p>S1.1</p></td>
<td><p>PA03.09, PA03.11, PA08.01.01, PA10.01, PA11.01</p></td>
<td><p>L.2.2.1</p></td>
<td><p>Sending samples to the technician known to be researching a specific disease could link the samples to that disease</p></td>
<td><p>Predictability</p>
<p>Disassociability</p>
</td>
</tr>
<tr class="row-even"><td><p>15</p></td>
<td><p>S1.1</p></td>
<td><p>PA03.09, PA03.11, PA08.01.01, PA10.01, PA11.01</p></td>
<td><p>L.2.2.1</p></td>
<td><p>Sending samples to the wet lab known to be researching a specific disease at that time could link the samples to that disease</p></td>
<td><p>Predictability</p>
<p>Disassociability</p>
</td>
</tr>
<tr class="row-odd"><td><p>26</p></td>
<td><p>S6</p></td>
<td><p>PA05.01.01</p></td>
<td><p>I.2.1.1</p></td>
<td><p>Nature of genomic data makes complete disassociability impossible to guarantee</p></td>
<td><p>Predictability Disassociability</p></td>
</tr>
<tr class="row-even"><td><p>55</p></td>
<td><p>S6</p></td>
<td><p>PA03.09, PA09.01.01, PA09.01.03, PA09.01.04, PA11.01</p></td>
<td><p>DD.4.1.2</p></td>
<td><p>Bioinformatics tools come from a variety of developers that can change over time; corruption within this supply chain, especially if left unmonitored, could result in research subject data being disclosed</p></td>
<td><p>Predictability</p></td>
</tr>
<tr class="row-odd"><td><p>65</p></td>
<td><p>S17</p></td>
<td><p>PA02.02, PA07.05</p></td>
<td><p>U.1.1</p></td>
<td><p>Data subject does not clearly understand what data actions that analysis tools along the pipeline will perform on their data</p></td>
<td><p>Predictability Manageability</p></td>
</tr>
</tbody>
</table>
</div>
<p>To understand the validation process, consider attack number 14 as a specific example from Table 15. The PANOPTIC threat actions and sub-actions that make up the attack map to the LINDDUN threat types of Linking, Non-repudiation, Detecting, and Data Disclosure. (Definitions of these are provided in <a class="reference external" href="Appendix/appendixC.html">Appendix C</a>.) Neither Non-repudiation nor Detecting is relevant to this scenario and can be dropped from consideration. By sorting the dataflow analysis table (Table 12) on the LINDDUN threat designators it is then possible to review the dataflows related to Linking and Data Disclosure. Matching scenario components are then identified by sorting on the Dataflow column to group those entries involving physical samples[13]_. The dataflow analysis for the core example contains multiple instances involving physical samples susceptible to threat L2.2.1, profiling an individual. This validates attack 14 against the LINDDUN analysis. Based on both the LINDDUN threat and the PANOPTIC threat actions (profiling and revelation in particular), attack 14 clearly undermines predictability as well as disassociability, validating it against the PEOs. Therefore, we can conclude that this is a valid threat.</p>
<p>As Table 15 indicates, all PANOPTIC attacks were successfully validated against LINDDUN threats and the LINDDUN-supported attacks validated against the PEOs. As a result, all the threats are candidates for responses.</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id6" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>12<span class="fn-bracket">]</span></span>
<p>While the diagram organizes the nodes by data action, dataflow segments may involve more than a single data action.</p>
</aside>
<aside class="footnote brackets" id="id7" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id4">13</a><span class="fn-bracket">]</span></span>
<p>In the core example the number of dataflows and associated threats is so limited that no sorting is necessary. In contrast, the complete example contains almost 100 itemized LINDDUN threats.</p>
</aside>
</aside>
</section>
</section>


                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="Question1.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Question 1: “What are we working on?”</p>
      </div>
    </a>
    <a class="right-next"
       href="Question3.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Question 3: “What are we going to do about it?”</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#linddun-analysis">LINDDUN Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#panoptic-analysis">PANOPTIC Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#threat-validation">Threat Validation</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<p>
  
    By NIST<br/>
  
</p>

          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>
<footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        <div class="disclaimer">
          <strong>Disclaimer:</strong> Certain commercial equipment, instruments, or
          materials are identified in this documentation to foster understanding. Such
          identification does not imply recommendation or endorsement by the National
          Institute of Standards and Technology, nor does it imply that the materials
          or equipment identified are necessarily the best available for the purpose.
        </div>
      </p>
    </div>
</footer>
  </body>
</html>